packages/p2p-v2/src/pages/buy-sell/screens/BuySellHeader/__tests__/BuySellHeader.spec.tsx

```diff
@@ -121,4 +126,14 @@ describe('<BuySellHeader />', () => {
 
         expect(mockProps.setIsFilterModalOpen).toHaveBeenCalledWith(true);
     });
+
+    it('should allow users to click on filter button', () => {
+        render(<BuySellHeader {...mockProps} />);
+
+        const filterButton = screen.getByTestId('dt_p2p_v2_buy_sell_header_filter_button');
```

Comment:
why not target the button instead of the icon?

Reply:
its actually not a button but an Icon 😅 and its hard to target the button because we would need to use queryByRole, then target the button in the array of Buttons, since we can't target the icon